runtime/src/accounts_db.rs

```diff
@@ -8539,7 +8539,10 @@ impl AccountsDb {
         let slots = self
             .storage
             .iter()
-            .map(|k| *k.key() as Slot)
+            .filter_map(|k| {
+                let slot = *k.key() as Slot;
+                requested_slots.contains(&slot).then_some(slot)
+            })
```

Comment:
```suggestion
            .map(|k| *k.key() as Slot)
            .filter(|slot| requested_slots.contains(&slot))
```
a bit clearer imo, and smaller diff 😄

Reply:
ah, I think I broke the checks Jeff - I forgot to remove the & from slot on the added line